src/components/TrackTile/services/TrackTileService.ts

```diff
@@ -139,23 +139,11 @@ export type TrackTileService = {
   readonly patientId?: string;
 
   /**
-   * @description account and project that the patient data will be saved to
-   */
-  readonly datastoreSettings: {
-    readonly account: string;
-    readonly project: string;
-  };
-
-  /**
-   * @description account and project that is used for fetching tracker settings. Defaults to **datastoreSettings**
+   * @description account and project that is used for fetching tracker settings.
    */
   readonly accountSettings?: {
```

Comment:
Might be nice to remove this and hoist account and project up to the service level.